core-initrd/latest/factory/usr/lib/udev/rules.d/90-ubuntu-core-partitions.rules

```diff
@@ -0,0 +1,15 @@
+SUBSYSTEM!="block", GOTO="ubuntu_core_partitions_end"
+
+ENV{DEVTYPE}=="partition", IMPORT{parent}="UBUNTU_*_UUID"
+IMPORT{program}="/usr/lib/snapd/snap-bootstrap scan-disk"
+ENV{DEVTYPE}=="disk", ENV{UBUNTU_DISK}=="1", SYMLINK+="ubuntu/disk"
+ENV{DEVTYPE}=="partition", ENV{UBUNTU_SEED}=="1", SYMLINK+="ubuntu/seed"
+ENV{DEVTYPE}=="partition", ENV{UBUNTU_BOOT}=="1", SYMLINK+="ubuntu/boot"
+ENV{DEVTYPE}=="partition", ENV{UBUNTU_DATA}=="1", ENV{ID_FS_TYPE}=="crypto_LUKS", SYMLINK+="ubuntu/data-luks"
+ENV{DEVTYPE}=="partition", ENV{UBUNTU_DATA}=="1", ENV{ID_FS_TYPE}!="crypto_LUKS", SYMLINK+="ubuntu/data"
+ENV{DEVTYPE}=="partition", ENV{UBUNTU_SAVE}=="1", ENV{ID_FS_TYPE}=="crypto_LUKS", SYMLINK+="ubuntu/save-luks"
+ENV{DEVTYPE}=="partition", ENV{UBUNTU_SAVE}=="1", ENV{ID_FS_TYPE}!="crypto_LUKS", SYMLINK+="ubuntu/save"
```

Comment:
Should we use better `/dev/disk/ubuntu/`? I would prefer this a bit more hidden.